packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -52,6 +52,18 @@ You can create up to three custom domains with multibrand customizations and up
 
 You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) after you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
 
+#### Redirect URIs
+
+Multibrand orgs use dynamic issuer mode for IdP. As a result, Okta uses the domain from the authorize request as the domain for the redirect URI when returning the authentication response. The Admin Console UI displays the org's Okta subdomain when the org has multiple custom domains configured.
```

Comment:
I understood the acronym "IdP" as "Identity Provider", which doesn't really work in this sentence. Is it correct? (Applies to previous section as well)

Reply:
I'll dig and get more info. Thanks!